content/api/editor-support.md

```diff
@@ -4,26 +4,26 @@ title: Editor support
 
 Installing our code locally will allow your editor to extract information from our code, giving you additional features like auto-complete (also known as [IntelliSense](https://en.wikipedia.org/wiki/IntelliSense)).
 
-Installing these definitions is optional, however will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
+Installing these definitions is optional, but will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
 
 ## Installation
-Installing our API, the code you interact with, can be installed using `pip`, Python's built-in package manager. `pip` should be installed on your machine, if it's not, you'll need to look up how to install it. See [here](https://stackoverflow.com/a/12476379) for troubleshooting on Windows.
+Our API, the code you interact with, can be installed using Python's built-in package manager `pip`. Assuming you have Python installed on your machine, `pip` should be installed as well. If it's not, you'll need to look up how to install it. See [here](https://stackoverflow.com/a/12476379) for troubleshooting on Windows.
```

Comment:
`pip` is _not_ installed by default on Ubuntu... though fixing that feels out of scope for this PR.

Reply:
Oops, wasn't really thinking about the meaning of this paragraph when I reworded it. Have added a mention that this is Windows-specific behaviour.